modified:pom.xml

JustaMiracle opened the pull request:
<groupId>com.squareup.okhttp3</groupId>
            <artifactId>okhttp</artifactId>
	    a<version>3.14.2</version>

if the version of okhttp is 4.9.1, it will  possibly not be able to import into the Maven project,
my version of maven is  3.5.2, in this version , the import okhttp will fail;
so if using the version 3.14.2,it  will be better.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/casdoor/casdoor-java-sdk?pullRequest=19) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/casdoor/casdoor-java-sdk?pullRequest=19) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/casdoor/casdoor-java-sdk?pullRequest=19) it.</sub>

hsluoyz commented:
@seriouszyx

seriouszyx commented:
@JustaMiracle 
Hi, as far as I know, the version 4.x of okhttp used kotlin to implement and there are some [Backwards-Incompatible Changes](https://square.github.io/okhttp/changelogs/upgrading_to_okhttp_4/#backwards-incompatible-changes). Some bugs are also fixed in the new version. So why must we downgrade the okhttp version?
I tested version 3.14.2 and it has no effect on the current unit tests. But some other Casdoor libs, such as casdoor-spring-boot-starter, rely on casdoor-java-sdk for implementation. I'm not sure whether this PR will have an impact on other libs (for example, the compatibility between okhttp and springboot). 
And I tested with maven 3.5.2, which also can import okhttp4.9.1. Can you provide the error message, or upgrade your version of maven? That should be easy.

JustaMiracle commented:
> @JustaMiracle Hi, as far as I know, the version 4.x of okhttp used kotlin to implement and there are some [Backwards-Incompatible Changes](https://square.github.io/okhttp/changelogs/upgrading_to_okhttp_4/#backwards-incompatible-changes). Some bugs are also fixed in the new version. So why must we downgrade the okhttp version? I tested version 3.14.2 and it has no effect on the current unit tests. But some other Casdoor libs, such as casdoor-spring-boot-starter, rely on casdoor-java-sdk for implementation. I'm not sure whether this PR will have an impact on other libs (for example, the compatibility between okhttp and springboot). And I tested with maven 3.5.2, which also can import okhttp4.9.1. Can you provide the error message, or upgrade your version of maven? That should be easy.



> @JustaMiracle Hi, as far as I know, the version 4.x of okhttp used kotlin to implement and there are some [Backwards-Incompatible Changes](https://square.github.io/okhttp/changelogs/upgrading_to_okhttp_4/#backwards-incompatible-changes). Some bugs are also fixed in the new version. So why must we downgrade the okhttp version? I tested version 3.14.2 and it has no effect on the current unit tests. But some other Casdoor libs, such as casdoor-spring-boot-starter, rely on casdoor-java-sdk for implementation. I'm not sure whether this PR will have an impact on other libs (for example, the compatibility between okhttp and springboot). And I tested with maven 3.5.2, which also can import okhttp4.9.1. Can you provide the error message, or upgrade your version of maven? That should be easy.

ok，maybe i should upgrade my version of maven

JustaMiracle commented:
> @JustaMiracle Hi, as far as I know, the version 4.x of okhttp used kotlin to implement and there are some [Backwards-Incompatible Changes](https://square.github.io/okhttp/changelogs/upgrading_to_okhttp_4/#backwards-incompatible-changes). Some bugs are also fixed in the new version. So why must we downgrade the okhttp version? I tested version 3.14.2 and it has no effect on the current unit tests. But some other Casdoor libs, such as casdoor-spring-boot-starter, rely on casdoor-java-sdk for implementation. I'm not sure whether this PR will have an impact on other libs (for example, the compatibility between okhttp and springboot). And I tested with maven 3.5.2, which also can import okhttp4.9.1. Can you provide the error message, or upgrade your version of maven? That should be easy.

ok，maybe i should upgrade my version of maven
